hera_cal/abscal_funcs.py

```diff
@@ -1563,7 +1462,7 @@ def avg_file_across_red_bls(data_fname, outdir=None, output_fname=None,
         uvd.read_miriad(data_fname)
 
     # get data
-    data, flags = UVData2AbsCalDict(uvd)
+    data, flags = io.load_vis(uvd, pop_autos=True, return_meta=False, pick_data_ants=True)
```

Comment:
You don't need return_meta=False.

Reply:
yeah, just in case defaults change doesn't hurt to have